docs/content/assets/css/customizations.css

```diff
@@ -50,27 +50,35 @@ div.md-content article.md-content__inner a.toclink {
   color: currentColor;
 }
 
-/* Positions the permalink symbol to the left of the heading for consistent placement */
-/* Due to position order of the anchor link element to the heading content, it was better to replace it */
-/* Changed from a symbol to a solid thin rectangle due to vertical alignment and scaling (clipping) issues */
-div.md-content article.md-content__inner a.toclink::before {
-  display: inline-block;
-  margin-right: 0.2rem;
-  margin-left: -0.6rem;
-  content: '-'; /* not visible, just used as filler to make background-color visible */
-  background-color: #73a4d7; /* light blue, could modify theme colour vars to match this? */
-
-  /* Based on original `.headerlink` styles from `mkdocs-material` theme: */
-  color: transparent;
-  opacity: 0;
-  transition: opacity 125ms;
+/* Instead of a permalink symbol at the end of heading text, use a border line on the left spanning height of heading */
+/* Includes optional background fill with rounded right-side corners, and restores inline code style */
+/* NOTE: Headings with markdown links embedded disrupt the bg fill style, as they're not children of `a.toclink` element */
+div.md-content article.md-content__inner a.toclink {
+  display: inline-block; /* Enables multi-line support for both border and bg color */
+  border-left: .2rem solid transparent; /* transparent placeholder to avoid heading shift during reveal transition */
+  margin-left: -0.6rem; /* Offset heading to the left */
+  padding-left: 0.4rem; /* Push heading back to original position, margin-left - border-left widths */
+  transition: background-color 200ms,border-left 200ms;
+
+  /* Only relevant if using background highlight style */
+  border-radius: 0 0.25rem 0.25rem 0;
+  padding-right: 0.4rem;
 }
 
-/* Based on original `.headerlink` styles from `mkdocs-material` theme: */
-div.md-content article.md-content__inner a.toclink:hover::before,
-div.md-content article.md-content__inner :target > a.toclink::before {
-  opacity: 1;
-  transition: opacity 125ms;
+div.md-content article.md-content__inner a.toclink:hover,
+div.md-content article.md-content__inner :target > a.toclink {
+  border-left: .2rem solid #448aff; /* highlight line on the left */
+  background-color: #b3dbff6e; /* background highlight fill */
```

Comment:
Maybe we change the blue color to the purple-ish that is used in the header?

Reply:
We could change the theme colour (`mkdocs.yml` config) or if you prefer the current colour scheme (I don't quite like the colour of links personally), I can use a theme variable and it can use that.

Both colours here are blue though. I liked the background highlight when viewing on mobile, but we can remove that if you prefer. Did you want the blue bar on the left to be changed? It was similar to the header colour I think.

Reply:
![image](https://user-images.githubusercontent.com/20406381/110912417-3499a080-8314-11eb-986f-61202c83c327.png)

There are 3 parts that get colorized:
1. the background of the header
2. the background and the vertical line of a clicked/selected headline
3. link in the text

1. and 3. have the same color but 2. uses blue instead of the purple-ish of 1. and 3. So I was wondering if we want to change the color of 2. to the color of 1. and 3.

Reply:
Neither of the theme colours mentioned would be an appropriate substitute for the background highlight colour, they're too dark it would reduce readability of the heading text.

The darker blue line on the side of a heading can use the theme colour sure. Add it to the list of tasks you'd like done and I'll get on to it once I wrap up other tasks.